fla/ops/delta_rule/fused_recurrent.py

```
@@ -47,9 +47,9 @@ def fused_recurrent_delta_rule_fwd_kernel(
     if IS_VARLEN:
         bos, eos = tl.load(cu_seqlens + i_n).to(tl.int64), tl.load(cu_seqlens + i_n + 1).to(tl.int64)
         all = T
```

Ruff (0.15.9)

[error] Variable `all` is shadowing a Python builtin

(A001)